Code Review: Add docs for add-library scheme command (#677) (#678)
sketch-hq/SketchAPI: docs/pages/app.md

**Note:** If a document is already open the canvas view does not change.

### Table of Contents

- [Center layer and specify zoom level](#centerlayer)
- [Run a plugin command](#runplugincommand)

> - [Add a Remote Library](#addremotelibrary)

Reviewer: Looking at the main Sketch documentation, maybe using the term shared Library is better overall

```suggestion
- [Add a shared Library](#add-shared-library)
```

Author: Changed.

> ## Center layer and specify zoom level <a name="centerlayer"></a>

Reviewer: ```suggestion
## Center layer and specify zoom level <a name="center-layer"></a>
```

_Actual Size_ has a zoom level of `1`. To zoom out specify values between `0...1` while using values greater than `1` to zoom in, e.g. `2` for 200%.

> ## Run a plugin command <a name="runplugincommand"></a>

Reviewer: ```suggestion
## Run a plugin command <a name="run-plugin-command"></a>
```

Author: Changed.

Open the following link to display a _Hello World_ message within an open document. Provide any URL encoded string with the `msg` query parameter.

[`sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World`](sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World)

> ## Add a Remote Library <a name="addremotelibrary"></a>

Reviewer: ```suggestion
## Add a shared Library <a name="add-shared-library"></a>
```

Author: Changed.

> You can add a Remote Library to Sketch by providing a url to an appcast file.

Reviewer: I'd make it more explicit that it's about libraries hosted somewhere using RSS. Also for consistency with the other entries, please make sure to use identical formatting of variables and parameter names, plus put the main URL scheme command straight below the summary paragraph. (don't accept suggestion verbatim, indention of snippet isn't correct but required for the suggestion not to break).

```suggestion
Add a Library hosted on any server and shared using RSS by specifying the URL to the appcast.

    ```
    sketch://add-library?url=APPCAST_RSS_URL
    ```

To find out more about sharing your own libraries, see:

- [Sharing a Library](https://www.sketch.com/docs/libraries/#sharing-libraries) Sketch documentation
- [Appcast documentation](https://sparkle-project.org/documentation/)
```

Author: Changed.

> ### Parameters

Reviewer: ```suggestion
```

Author: Changed.

> Provide an `encoded appcast url` parameter to the `add-library` command.

Reviewer: ### `url` string

URL-encoded string specifying the address of the appcast RSS file.

Author: Changed.

```sh
sketch://add-library?url=AppCastURL
```

#### Example

> Open the link below to add a Remote Library to Sketch

Reviewer: Good example but it's not link, so instead describe _what_ it is:

```suggestion
URL to install the Apple iOS Sketch Library from [Apple Design Resources](https://developer.apple.com/design/resources/).
```

Author: Changed.

```sh
sketch://add-library?url=https%3A%2F%2Fdeveloper.apple.com%2Fdesign%2Fdownloads%2Fsketch.rss
```

## Related resources

- [Sharing a Library](https://www.sketch.com/docs/libraries/#sharing-libraries)

> - [Appcast documentation](https://sparkle-project.org/documentation/)

Reviewer: Let's drop this. Both links are very specific to the `add-library` command where both links are included now.
